package.json

```diff
@@ -9,8 +9,13 @@
     "serve": "npx backlight@latest serve 4BeMe20hqOWTkdUL2NuJ --open"
   },
   "dependencies": {
+    "@divriots/starter-simba": "^0.3.0",
```

Comment:
There is an opportunity here to create a dark/light switch web component that we open-source.
Could go in `dockit-core` so everyone can use it in their doc or even in webcomponent.dev as a standalone component. `dark/light` switch out of the box.

this one is a nice example we there are plenty of others
https://codepen.io/bheberer/pen/BaNZKmq

Not blocking